proto/node/v1/node.proto

```diff
@@ -93,6 +93,10 @@ message GovernanceMessage {
     WormchainInstantiateContract wormchain_instantiate_contract = 15;
     WormchainMigrateContract wormchain_migrate_contract = 16;
 
+    // Gateway
+    GatewayScheduleUpgrade gateway_schedule_upgrade = 24;
```

Comment:
Is there a reason we're skipping 22 and 23?

Reply:
To keep it consistent with the proposed changed to main in #3292